packages/jest-config/src/normalize.js

```diff
@@ -492,6 +492,11 @@ export default function normalize(options: InitialOptions, argv: Argv) {
       case 'watchman':
         value = options[key];
         break;
+      case 'watchPlugins':
+        value = (options[key] || []).map(watchPlugin =>
+          _replaceRootDirTags(options.rootDir, watchPlugin),
```

Comment:
How about using `resolve()` from `jest-config/utils.js`? We already do that when normalizing other options
https://github.com/facebook/jest/blob/5e903c1f31d8db8d914b7ae84830ad90c73061fc/packages/jest-config/src/utils.js#L31-L48

Reply:
yeah I'm happy to use that one instead :) 

Options:

1) resolve in `jest-cli` and get import from `build`
```js
import { resolve } from 'jest-config/build/utils'
```

2) do resolve in `jest-cli` and export `resolve` from `jest-config`
```js
import { resolve } from 'jest-config'
```

3) move `watchPlugin` resolution logic out of jest-cli and make it part of `normalize.js`
```js
      case 'watchPlugins':
        value = (options[key] || []).map(watchPlugin =>
          resolve(options.rootDir, key, watchPlugin),
        );
        break;
```

I think I prefer number 3

Reply:
Although number 3 means that watchPlugins need to be resolved even if it is not in watch mode

Reply:
I was thinking of option 3 specifically :)

Reply:
I guess the overhead will be minimal, because it's like - 0 to maybe 5 more paths to resolve. But I'm also ok with importing resolve from `jest-config`, since we already use it as dependency of `jest-cli`.

Reply:
Yep, let's do option three :)